src/store/sagas/handlers/user.js

```diff
@@ -29,14 +26,23 @@ export function* handleRegisterUser(action) {
     try {
 
         yield call(requestRegisterUser(action.userData))
+        
+    } catch (err) {
         const alertData = {
-            msg : "Successfully Registered",
-            alertType : "success",
+            msg : err.code,
+            alertType : 'error',
             id: uuidv4()
         }
-        console.log(alertData)
+        console.log(alertData, err.message)
         yield put(setAlert(alertData))
         yield put(enableButton())
+    }
+}
+
+export function* handleLoginUser(action) {
+    try {
+
+        yield call(requestLoginUser(action.userData))
```

Comment:
THere should be a `bind` method on functions that takes in parameter, to make it work without any error or warning.

Reply:
Please explain...

Reply:
Thanks .... Already added this... that took care of the error I was getting even on success.

Reply:
Yhhh.

Reply:
Do you feel this can be approved now?